src/Entities/AllocationCollection.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * This source file is proprietary and part of Rebilly.
+ *
+ * (c) Rebilly SRL
+ *     Rebilly Ltd.
+ *     Rebilly Inc.
+ *
+ * @see https://www.rebilly.com
+ */
+
+namespace Rebilly\Entities;
+
+use Rebilly\Rest\Entity;
+
+/**
+ * Class AllocationCollection.
+ */
+final class AllocationCollection extends Entity
+{
+    /**
+     * @param array $data
+     *
+     * @return AllocationCollection
+     */
+    public static function createFromData(array $data)
+    {
+        return new self($data);
+    }
+
+    /**
+     * @return TransactionAllocation[]
+     */
+    public function getTransactions()
+    {
+        return $this->getAttribute('transactions');
+    }
+
+    /**
+     * @param TransactionAllocation[] $value
+     *
+     * @return $this
+     */
+    public function setTransactions($value)
+    {
+        return $this->setAttribute('transactions', $value);
+    }
+
+    /**
+     * @param array $data
+     *
+     * @return array|TransactionAllocation[]
+     */
+    public function createTransactions(array $data)
+    {
+        return array_map(function ($item) {
```

Comment:
```suggestion
        return array_map(static function ($item) {
```

Reply:
I can pick this up as a separate PR for the whole project.

Reply:
Please leave it without static.

Reply:
May I know why @ashkarpetin?

Reply:
and also why don't we use return type declarations?